Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis.yml

```diff
@@ -379,9 +442,9 @@ view: |-
     },
     "paper": {
       "dimensions": {
-        "height": 1455,
-        "width": 740,
-        "x": 1110,
+        "height": 1545,
+        "width": 1230,
+        "x": 1450,
         "y": -20
       }
     }
```

Comment:
It doesn't looks like you added the error_msg output to the playbook in this PR.